docs/src/documentation/05-development/01-guides/04-no-exporting-components.mdx

```diff
@@ -77,14 +77,14 @@ and would require a lot of work in users' repositories.
 As far as we know, there isn't any tooling that would allow us to control or determine
 which CSS properties are being used in the extended component and which are being overridden.
 So we couldn't transparently support how our components would be used.
-This, in the end, could lead to inconsistent behaviour or visual style for our applications,
+This, in the end, could lead to inconsistent behavior or visual style for our applications,
 even though we can't prohibit all possibilities when it comes to custom CSS overrides.
 
-### Extending functionality to styled-components requires running more JS code
+### Extended functionality requires running more JS code
 
 Given the nature of how styled-components work,
 allowing extension could lead to unnecessary complexity in UI components
-and also more time-consuming runtimes.
+and also more time-consuming run times.
```

Comment:
"Runtime" is a valid programming term, although I'm not sure about the use of plural.

Reply:
Didn't seem like it in plural, so I opted for something that passed. But I can shift it back to singular.